redis/src/main/scala/zio/redis/TestExecutor.scala

```diff
@@ -2087,11 +2088,60 @@ private[redis] final class TestExecutor private (
                   })
                 else
                   Chunk.fromIterable(unionMap.toArray.sortBy(_._2).map(e => bulkString(e._1)))
-                  
+
             } yield RespValue.Array(result)
           )
         )
 
+      case api.SortedSets.ZRandMember =>
+        val key              = input(0).asString
+        val maybeCount        = input.tail.headOption.map(b => b.asString.toLong)
+        val withScoresOption = input.map(_.asString).find(_ == "WITHSCORES")
+
+        orWrongType(isSortedSet(key))(
+          for {
+            scoreMap <- sortedSets.getOrElse(key, Map.empty)
+            asVector = scoreMap.toVector
+            res <- maybeCount match {
+              case None => selectOne(asVector, randomPick).map {
```

Comment:
Is `selectOne` method provided by ZIO?

Reply:
@jxnu-liguobin no it's defined in the same file on line 2229